ci/constraints-test/py311-min-install.txt

```diff
@@ -0,0 +1,3 @@
+numpy==1.25.2
```

Comment:
Going to decrease this until it starts failing.

Will merge once I find the min version.

Reply:
`1.23` seems to be the oldest version of `numpy` that works on `3.11`